### DISCUSSION TITLE
How can I set up buying items in my store?

### DISCUSSION THREAD

**Discussion by rebeccakzpt** [naverz/zepeto-studio-global#6] [Q&A]
Has anybody done this before? I have a lot of questions...

**Comment by jhslee** [comment]
Hello again~ 

I see your question was answered [here](https://github.com/naverz/zepeto-studio-kor/discussions/715)
Its unfortunate that we support purchasing your personal creator items 😢, but please rest assured that we do have plans to implement this feature going forward. 

The global thread has been silent for a while, but I'm here to answer any questions you might have in the future, so please don't hesitate to ask~
